pagopa/io-app
# chore: [IOBP-387] Disable scan feature if barcode screen is loading

## Short description
This PR adds disabling of the barcode scan screen if it's currently loading.

## List of changes proposed in this pull request
- [ts/features/barcode/screens/BarcodeScanScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-387-disable-scan-if-screen-is-loading?expand=1#diff-12570b34029e534a64b24f1cb81383d99557273c4b2f34ad4ced82685fa017f6): added `isFileReaderLoading` to `isDisabled` prop evalutation
[ts/features/idpay/payment/screens/IDPayPaymentCodeScanScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-387-disable-scan-if-screen-is-loading?expand=1#diff-eaf783c2cc9816aca220690082f4ba077df5954553a99078eb5e3faec9bd4b27): added `isFileReaderLoading` to `isDisabled` prop evalutation
[ts/features/walletV3/payment/screens/WalletPaymentBarcodeScanScreen.tsx](https://github.com/pagopa/io-app/compare/IOBP-387-disable-scan-if-screen-is-loading?expand=1#diff-627f530afb8e81f8849fbfbfa2ee5ece0434bdbe062355178c9df587151a9513): added `isFileReaderLoading` to `isDisabled` prop evalutation

## How to test
Navigate to the barcode scan screen and check that the scan feature is disabled during the file/image decoding.
